RubberduckTests/QuickFixes/RemoveEmptyConditionBlockQuickFixTests.cs

```diff
@@ -556,13 +562,16 @@ End If
             var vbe = MockVbeBuilder.BuildFromSingleStandardModule(inputCode, out component);
             var state = MockParser.CreateAndParse(vbe.Object);
 
-            var inspection = new EmptyIfBlockInspection(state);
+            var inspection = new EmptyConditionBlockInspection(state);
             var inspector = InspectionsHelper.GetInspector(inspection);
             var inspectionResults = inspector.FindIssuesAsync(state, CancellationToken.None).Result;
 
-            new RemoveEmptyIfBlockQuickFix(state).Fix(inspectionResults.First());
+            //Skip over empty Else block and get If block for the fix
+            var inspectionToFix = inspectionResults.Skip(1).First();
```

Comment:
Update with `ElementAt(1)` to replace `Skip(1).First()` for slight improvement.

Reply:
With `if(inspectionResults.Count() < 2) { Assert.Inconclusive("Test code did not generate expected inspection results"); }` the test would not attempt to fix a non-existent result - I think a quickfix test needs to be inconclusive if the inspection doesn't work as it should. That way you don't get distracted by a failing quickfix test when the real problem is with the inspection (which should then have at least one failing test).